info_terminal/infremen/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(infremen)
+
+find_package(catkin REQUIRED COMPONENTS  mongodb_store roscpp genmsg actionlib_msgs actionlib) 
+
+set ( CMAKE_CXX_FLAGS "-Ofast -march=native -mno-avx -ggdb")
+
+generate_messages(
+	#   DEPENDENCIES actionlib_msgs 
+)
+
```

Comment:
Since it doesn't have custom messages, actions, or services it shouldn't call the `generate_messages` macro and also should not depend on `genmsg`, `message_runtime` or `message_generation`. In general `genmsg` should be substituted with `message_generation`.